civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5025,7 +5031,9 @@ function _civihr_employee_portal_should_do_onboarding($account) {
   $canViewOnboardingForm = node_access('view', $onboardingWebform, $account);
   $submissions = WebformHelper::getUserSubmissionsByTitle($account, $formName);
 
-  return $canViewOnboardingForm && empty($submissions);
+  return
```

Comment:
I think this conditional is getting too long, can you separate it into something else like 

```
$canCustomizeWizard = user_access('customize welcome wizard');
$isAdminTest = $userWantsToTest && $canCustomizeWizard;
```

Reply:
Right, thanks!